app/templates/components/code-module.hbs

```diff
@@ -10,27 +9,25 @@
       {{/if}}
     </div>
 
-    <div class="row" style="margin-top:20px; margin-left:0px; margin-right:20px">
-        <label>Editor kódu</label>
-        {{code-editor content=module.code id=(construct-id "editor_module" module.id)}}
-    </div>
+    <label>Editor kódu</label>
+    {{code-editor content=module.code id=(construct-id "editor_module" module.id)}}
 
-    <div class="row hidden-print" style="margin-left:0px; margin-right:20px">
-        <button class="btn btn-primary btn-task inline {{if (or running submitting) "disabled"}}" style="margin-top:20px;" {{action "run"}}>{{if running "Spouštím..." "Spustit"}}</button>
-        <button class="btn btn-primary btn-task inline {{if (or running submitting) "disabled"}}" style="margin-top:20px;" {{action "load"}}>Nahrát soubor z disku</button>
-        <button class="btn btn-primary btn-task inline" style="margin-top:20px;" {{action "save"}}>Uložit soubor na disk</button>
-        <button class="btn btn-primary btn-task inline {{if (or running submitting) "disabled"}}" style="margin-top:20px;" {{action "reset"}}>Obnovit výchozí kód</button>
+    <div class="hidden-print mtop15">
+        <button class="btn btn-primary btn-task inline mtop5 {{if (or running submitting) "disabled"}}" {{action "run"}}>{{if running "Spouštím..." "Spustit"}}</button>
+        <button class="btn btn-primary btn-task inline mtop5 {{if (or running submitting) "disabled"}}" {{action "load"}}>Nahrát soubor z disku</button>
+        <button class="btn btn-primary btn-task inline mtop5" {{action "save"}}>Uložit soubor na disk</button>
+        <button class="btn btn-primary btn-task inline mtop5 {{if (or running submitting) "disabled"}}" {{action "reset"}}>Obnovit výchozí kód</button>
         {{input type="file" class="hide" id=(construct-id "load_input" module.id) accept=".py"}}
         {{#if session.current_user.organisator}}
-            {{#link-to "admin/execs" (query-params module=module.id) class="btn btn-primary btn-task inline mtop20"}}→ Spuštění{{/link-to}}
+            {{#link-to "admin/execs" (query-params module=module.id) class="btn btn-primary btn-task inline mtop5"}}→ Spuštění{{/link-to}}
         {{/if}}
     </div>
 
     {{ale-rt type="danger" show=show_error closeBtn=true text=general_error style="margin-top:20px"}}
     {{ale-rt type=script_message_mode show=show_message closeBtn=true text=script_message_output style="margin-top:20px"}}
     {{#if show_load_info}}
         {{#if module.last_datetime}}
-            {{ale-rt type="info" closeBtn=true text=(concat "Byl načten kód z posledního " module.last_origin_text " (" (date-and-time module.last_datetime 'cs') "), pro obnovení původního kódu stiskni tlačítko Obnovit výchozí kód.")}}
+            {{ale-rt type="info" closeBtn=true style="margin-top:20px" text=(concat "Byl načten kód z posledního " module.last_origin_text " (" (date-and-time module.last_datetime 'cs') "), pro obnovení původního kódu stiskni tlačítko Obnovit výchozí kód.")}}

```

Comment:
Tohle je asi teda _mtop20_. Měnit to ale asi není třeba, je toho skrz kódy myslím víc. Spíš teda asi vytvořit issue.